audits/mobile-friendly/display.js

```diff
@@ -55,10 +55,13 @@ class ManifestDisplay extends Audit {
    */
   static audit(artifacts) {
     const manifest = artifacts.manifest.value;
+    const displayValue = (!manifest || !manifest.display) ? undefined : manifest.display.value;
+
+    const hasRecommendedValue = ManifestDisplay.hasRecommendedValue(displayValue);
 
     return ManifestDisplay.generateAuditResult(
-      ManifestDisplay.hasRecommendedValue(manifest.display.value),
-      manifest.display.value,
+      hasRecommendedValue,
+      displayValue,
```

Comment:
the manifest parser also has a debug string for display value if it isn't one of `fullscreen`, `standalone`, `minimal-ui`, or `browser` it might be worth piping through. Not sure the best way to combine it with this warning, though.